Reconstruction/PFA/Pandora/GaudiPandora/src/CaloHitCreator.cpp

```diff
@@ -431,8 +430,7 @@ pandora::StatusCode CaloHitCreator::CreateHCalCaloHits(const CollectionMaps& col
             {
                 try
                 {
-                    auto pCaloHit0 = pCaloHitCollection.at(i);
-                    auto pCaloHit = &(pCaloHit0);
+                    auto pCaloHit = const_cast<edm4hep::CalorimeterHit*>(&(pCaloHitCollection.at(i)));
```

Comment:
Why need to use the `const_cast`?

Reply:
You are right, now the const_cast is removed.